### ISSUE TITLE
Describe* links from doc-strings non-functional?

### ISSUE THREAD

**Issue by franburstall** [atlas-engineer/nyxt#2576]
**Describe the bug**

Not sure if this is a bug or a feature I don't understand.

Some links in describe-* pages lead to a dead end.

**Precise recipe to reproduce the issue**

Do `C-h C buffer` and click in the `buffer-delete` link in the documentation of `buffer-delete-hook`.

*What I expect*: some description of `buffer-delete`.

*What actually happens*: I get the dialog in this screenshot which is not easy to get out of (`C-g` does not always dismiss it) and conveys no information.

![1663263729](https://user-images.githubusercontent.com/5726790/190474090-110d8183-f144-4745-871e-0922d4b99535.png)

Observation: I get this behaviour for all links of the following form:
```
javascript:%28function%20%28%29%20%7B%0A%20%20%20%20var%20url%20%3D%20%27lisp%3A%2F%2F1739%2Fdescribe-any%27%3B%0A%20%20%20%20var%20request%20%3D%20fetch%28url%2C%20%7B%20%27mode%27%20%3A%20%27no-cors%27%20%7D%29%3B%0A%20%20%20%20if%20%28null%29%20%7B%0A%20%20%20%20%20%20%20%20__PS_MV_REG%20%3D%20%5B%5D%3B%0A%20%20%20%20%20%20%20%20return%20chain%28request%2C%20then%28function%20%28response%29%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20__PS_MV_REG%20%3D%20%5B%5D%3B%0A%20%20%20%20%20%20%20%20%20%20%20%20return%20at%28response%2C%20ok%29%20%3F%20chain%28response%2C%20json%28%29%29%20%3A%20null%3B%0A%20%20%20%20%20%20%20%20%7D%29%2C%20then%28null%29%29%3B%0A%20%20%20%20%7D%3B%0A%7D%29%28%29%3B
```
I think these links are built by `resolve-backtick-quote-links` and possibly shouldn't be?

**Information**
- OS name+version: debian-11
- Graphics card and driver:
- Desktop environment / Window manager name+version: exwm
- How you installed Nyxt (Guix pack, package manager, build from source): source
- Information from `show-system-information`:
```
Nyxt version: 3-pre-release-1-470-ge81aceba
Renderer version: GI-GTK
Operating system kernel: Linux 5.10.0-18-amd64
Lisp implementation: SBCL 2.1.1 (Dynamic space size: 3221225472)
Features: (:QUICKLISP :NYXT-GI-GTK :NYXT-GTK :WEBKIT2 :WEBKIT2-2.36
 :WEBKIT2-CORS-ALLOWLIST :WEBKIT2-PASTE-PLAINTEXT :WEBKIT2-TRACKING
 :WEBKIT2-MUTE :WEBKIT2-EMOJI :WEBKIT2-MEDIA :WEBKIT2-SANDBOXING :GTK-3-22
 :GTK-3-20 :GTK-3-18 :GTK-3-16 :GTK-3-14 :GTK-3-12 :GTK-3-10 :GTK-3-8 :GTK-3-6
 :GTK-3-4 :GTK :GDK-3-22 :GDK-3-20 :GDK-3-18 :GDK-3-16 :GDK-3-14 :GDK-3-12
 :GDK-3-10 :GDK-3-8 :GDK-3-6 :GDK-3-4 :CAIRO-1-10 :CAIRO-1-12 :GDK-PIXBUF
 :GLIB-2-30 :GLIB-2-32 :GLIB-2-34 :GLIB-2-36 :GLIB-2-38 :GLIB-2-40 :GLIB-2-42
 :GLIB-2-44 :GLIB-2-46 :GLIB-2-48 :GLIB-2-50 :GLIB-2-52 :GLIB-2-54 :GLIB-2-56
 :GLIB-2-58 :GLIB :NYXT-3.1 :NYXT-3 :CUSTOM-HASH-TABLE-NATIVE :SLYNK
 :PLUMP-UTF-32 :PARENSCRIPT :FSET-EXT-STRINGS :GLOBAL-VARS :DECLARE-TYPES
 :NAMED-READTABLES :SWANK :LPARALLEL :21BIT-CHARS :CLOSER-MOP :CL-PPCRE-UNICODE
 :CL-UNICODE :CL-PPCRE :CHUNGA :FLEXI-STREAMS :BORDEAUX-THREADS
 :LPARALLEL.WITH-CLTL2 :LPARALLEL.WITH-CAS :LPARALLEL.WITH-STEALING-SCHEDULER
 :SPLIT-SEQUENCE CFFI-FEATURES:FLAT-NAMESPACE CFFI-FEATURES:X86-64
 CFFI-FEATURES:UNIX :CFFI CFFI-SYS::FLAT-NAMESPACE ALEXANDRIA::SEQUENCE-EMPTYP
 :FAST-IO-SV :FAST-IO :CL-JSON-CLOS :CL-JSON :SBCL-USES-SB-ROTATE-BYTE
 CHIPZ-SYSTEM:GRAY-STREAMS :THREAD-SUPPORT :ASDF3.3 :ASDF3.2 :ASDF3.1 :ASDF3
 :ASDF2 :ASDF :OS-UNIX :NON-BASE-CHARS-EXIST-P :ASDF-UNICODE :X86-64 :GENCGC
 :64-BIT :ANSI-CL :COMMON-LISP :ELF :IEEE-FLOATING-POINT :LINUX :LITTLE-ENDIAN
 :PACKAGE-LOCAL-NICKNAMES :SB-LDB :SB-PACKAGE-LOCKS :SB-THREAD :SB-UNICODE
 :SBCL :UNIX)

ASDF version: 3.3.1
ASDF registries: (NYXT-SOURCE-REGISTRY ENVIRONMENT-SOURCE-REGISTRY USER-SOURCE-REGISTRY
                      USER-SOURCE-REGISTRY-DIRECTORY
                      DEFAULT-USER-SOURCE-REGISTRY SYSTEM-SOURCE-REGISTRY
                      SYSTEM-SOURCE-REGISTRY-DIRECTORY
                      DEFAULT-SYSTEM-SOURCE-REGISTRY)
Critical dependencies: (/home/fran/common-lisp/nyxt/_build/cl-cffi-gtk/gtk/cl-cffi-gtk.asd
 /home/fran/common-lisp/nyxt/_build/cl-gobject-introspection/cl-gobject-introspection.asd
 /home/fran/common-lisp/nyxt/_build/cl-webkit/webkit2/cl-webkit2.asd)


```

**Comment by Ambrevar** [comment]
The short answer is that `buffer-delete` is unexported (and it shouldn't).

The long answer is... We need to overhaul our description functions so that they display all symbols.  Why don't we leverage multiple sources for that?  I'm on it.

**Comment by aartaka** [comment]
@aadcg, I wouldn't close it that soon, because our `resolve-backtick-quote-links` still generates opaque `javascript:` links. But then, I'm currently working on fixing it so you can leave this closed.

**Comment by aartaka** [comment]
Fixed in c4532cfb96e6cd83a906f5722e93269f1e5f9ec1 :P

**Comment by aadcg** [comment]
I believe this was automatically closed since it was attached with PR #2579. Thanks for being extra careful here.
